runtime/doc/syntax.txt

```diff
@@ -4678,7 +4678,7 @@ font={font-name}					*highlight-font*
 	When setting the font for the "Normal" group, this becomes the default
 	font (until the 'guifont' option is changed; the last one set is
 	used).
-	The following only works with Motif and Athena, not with other GUIs:
+	The following only works with Motif not with other GUIs:
```

Comment:
I guess you're trying to keep this PR's scope small, but since you're already modifying this line you might as well delete it, IMO. Less work for future PRs.